### PR TITLE
Reader: update coldStartReader A/B test, enabling auto-follows

### PR DESCRIPTION
**Pull request by bluefuton**
Prior to the Cold Start experiment, we autofollowed new Readers to the "big 4": Daily Post, Discover, Longreads, and WordPress.com News.

This PR updates the experiment to re-introduce autofollowing, but otherwise keeps Cold Start the same.

Test live: https://calypso.live/?branch=add/reader/cold-start-v3

### CODE REVIEW

**Comment by bluefuton** [comment]
One potential tricky bit: we currently subtract 1 from the user’s total subscriptions to account for their own site. With the new version, we’d need to subtract 5 to account for their own site plus autofollows.

This obviously wouldn’t work for any existing users from CS v1 and v2 that get pushed back into Cold Start later because they’re following less than one site. They’d be following -4 sites and have to follow 5 to “graduate” :/

Two options:

- Remove the redirect that pushes existing users back into Cold Start later if they still haven’t followed anything. This is the easier option but would there be any impact on the experiment?
- Fix the counter to only take account of the current Cold Start session, ignoring the user’s subscription count. I aimed to fix this in https://github.com/Automattic/wp-calypso/pull/6526 but we decided not to change it for the original launch. Would need finishing up and reviewing.

**Comment by bluefuton** [comment]
^ Think we'll go with the "fix the counter" option. This means that #6526 must land as a prerequisite.

File: client/lib/abtest/active-tests.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
 		allowExistingUsers: true,
 	},
 	coldStartReader: {
-		datestamp: '20160804',
+		datestamp: '20160824',
 		variations: {
 			noEmailColdStart: 20,
```
Comment (gibrown) [inline client/lib/abtest/active-tests.js]: Can we run this as a 33-33-34 test? 33% on noEmailColdStart, 33% on EmailColdStart, 34% on noChanges?

That way we can compare them against each other, and we'll get to significant results faster.

Comment (bluefuton) [inline client/lib/abtest/active-tests.js]: Sounds like a plan 👍

Comment (bluefuton) [inline client/lib/abtest/active-tests.js]: @gibrown for the `EmailColdStart` variant, do you mean noEmailColdStart + autofollows?

Comment (gibrown) [inline client/lib/abtest/active-tests.js]: Yes, sorry, that's right

**Comment by bluefuton** [comment]
Closing in favour of @aaronyan's #7778.

**Comment by bluefuton** [comment]
Reopened so we can launch CS v3 separately to v4.

**Comment by bluefuton** [comment]
Todo:
- [ ] alter the Cold Start redirect limit depending upon A/B test

**Comment by blowery** [comment]
Do we need to readjust the start date on the abtest?

**Pull request merged**